src/components/main.js

```diff
@@ -25,29 +28,52 @@ export default function Header() {
     }
   };
 
+  const pickHex = (color1, color2, percent) => {
+    // Quash the gradient
+    if (percent < predThreshold) {
+      percent = percent / 3;
+    }
+    const w1 = percent / 100;
+    const w2 = 1 - w1;
+    var rgb = [
+      Math.round(color1[0] * w1 + color2[0] * w2),
+      Math.round(color1[1] * w1 + color2[1] * w2),
+      Math.round(color1[2] * w1 + color2[2] * w2)
+    ];
+    return rgb;
+  };
   const onFileChange = event => {
     setFile(event.target.files[0]);
   };
 
   const barGraphResults = results => {
     return results.map((res, index) => {
       let resToHeight = Math.round((res * 100) / 4) * 4;
-      // To display something at least
-      if (resToHeight === 0) {
-        resToHeight++;
-      }
-      // TODO: Update to full gradient
-      let barcolor = "green-200";
-      if (res > 0.25 && res < 0.5) {
-        barcolor = "green-400";
-      } else if (res >= 0.5 && res < 0.75) {
-        barcolor = "red-200";
-      } else if (res > 0.75) {
-        barcolor = "red-400";
+
+      const red = [238, 75, 43];
```

Comment:
Maybe we could refactor colors to constants file

Reply:
sure